example_snowpark/snowflake.yml

```
@@ -1,4 +1,4 @@
-definition_version: "1.1"
+<!! if snowflake_cli_version < "3.0.0" !!>definition_version: "1.1"
```

```suggestion
<!!- if snowflake_cli_version < "3.0.0" !!>
definition_version: "1.1"
```

Will this work? It should remove leading empty lines